### ISSUE TITLE
Missing file listing and file delete functionality from Edit Work

### ISSUE THREAD

**Issue by hannahfrost** [projecthydra-labs/hyku#530]
When editing an existing work containing file(s), I would expect to see the existing files listed on the Files tab, along with the ability to either delete an existing file or add more file(s). Instead all I see is the ability to add more files

![screenshot 2016-12-12 10 01 16](https://cloud.githubusercontent.com/assets/1801076/21110445/11a2e12e-c052-11e6-920f-41d6b2461a0f.png)

**Comment by mjgiarlo** [comment]
Related to this abandoned Sufia PR: https://github.com/projecthydra/sufia/pull/2026

**Comment by jcoyne** [comment]
@mjgiarlo how should we proceed with this?

**Comment by mjgiarlo** [comment]
@jcoyne should we get design before taking this forward?

**Comment by jcoyne** [comment]
@mjgiarlo I'm asking if we should take over/replace that sufia pr, or do you think they'll get back to that? It seems like @hannahfrost has some design in mind where the edit page shows the existing files. (which I think was what the Sufia UI group intended).

**Comment by mjgiarlo** [comment]
@jcoyne Yes, we should take over/replace that sufia PR. The person who was working on it no longer works for UVa, AFAIK. Good call, too -- we should roll w/ what the Sufia UI WG designed. I suspect we should do that work in Sufia (so the 7.x folks pick it up). Thanks!

**Comment by atz** [comment]
There's very little in that PR except for a table-driven template: 
```html
<% f.object.file_sets.each do |file| %>
   <tr>
       <td>
           <span><%= file.solr_document.label %></span>
       </td>
       <td>
           <span><%= file.title %></span>
       </td>
   </tr>
<% end %>
</tbody>
```

I don't think we need to use the existing PR at all (it's also marked closed).
